Sources/MockoloFramework/Parsers/SwiftSyntaxExtensions.swift

```diff
@@ -738,42 +751,77 @@ final class EntityVisitor: SyntaxVisitor {
     }
 
     override func visit(_ node: ImportDeclSyntax) -> SyntaxVisitorContinueKind {
-        if let ret = node.path.firstToken(viewMode: .sourceAccurate)?.text {
-            let desc = node.importKeyword.text + " " + ret
-            imports["", default: []].append(desc)
-        }
+        imports.append(
+            .init(line: node.trimmedDescription)
+        )
         return .skipChildren
     }
 
     override func visit(_ node: IfConfigDeclSyntax) -> SyntaxVisitorContinueKind {
-        for cl in node.clauses {
-            let macroName: String
+        let blockId = Self.ifConfigDeclCount
+        Self.ifConfigDeclCount += 1
+        var previousConditions: [String] = []
+
+        for (index, cl) in node.clauses.enumerated() {
+            let compilerDirectiveKey: String
```

Comment:
This `compilerDirectiveKey` does not seem to be just a key string, but rather a structured value. However, since it is currently a raw string, the structure it is meant to represent is unclear.  
I think it would be better to construct an `InsideDirective` directly.